libraries/mne/mne_hemisphere.h

```diff
@@ -227,7 +227,7 @@ inline bool operator== (const MNEHemisphere &a, const MNEHemisphere &b)
 {
     if(a.pinfo.size() == b.pinfo.size()) {
         for(int i = 0; i < a.pinfo.size(); ++i) {
-            if(!a.pinfo.at(i).isApprox(b.pinfo.at(i), 0.0001f)) {
```

Comment:
why was the precision for comparison removed?

Reply:
because it is not actually being used. the precision is casted to ze-ro. so i figured that if it hasn't been used all along... then we're sure we can live without it. Apart, I'm not sure 100%, but I think dose eigen people have somekind of precission implemented in their isapprox thing.

Reply:
https://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html#title33

Reply:
Sorry, I'm a bit dense, right now. Could you explain?

Following your previous comment, let me show you the actual warning that motivated this change.

![image](https://user-images.githubusercontent.com/8955424/78399298-462de080-75ba-11ea-852f-dc835e9e5c2d.png)

Reply:
ah sry `pinfo` is of type `Eigen::VectorXi`. In this case you are right, we are dealing with an integer value and the precision can be discarded.